src/data/admin/Mentors.tsx

```diff
@@ -14,39 +36,36 @@ export const TAGS = [
   },
 ];
 
-export const COLUMNS = [
+export const COLUMNS: extendedColumnDef<Person, string>[] = [
   generateSelect(),
   {
     accessorKey: "name",
     header: "Name",
     width: "w-3/12",
```

Comment:
can also use meta: { width: "" } and then change to column.columnDef.meta?.width in Table.jsx

Reply:
go for it, this change will also have to be propagated to the actual dashboards